### PR TITLE
feat(node-analyzer,kspm-collector,sysdig-deploy): allow custom proxy for individual containers in node analyzer

### PR DESCRIPTION
**Pull request by mavimo**
## Checklist

- [x] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [x] Chart Version bumped for the respective charts
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [x] All test files are added in the tests folder of their respective chart and have a "_test" suffix

### CODE REVIEW

File: charts/kspm-collector/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: kspm-collector
 description: Sysdig KSPM collector
 
-version: 0.8.5
+version: 0.9.5
```
Comment (mavimo) [inline charts/kspm-collector/Chart.yaml]:
```suggestion
version: 0.9.0
```

**Comment by dark-vex** [comment]
@mavimo I think I've figured out why `sysdig-deploy` unit-test was [failing](https://github.com/sysdiglabs/charts/actions/runs/6682922844/job/18158608556). 
It seems is because is not able to find the version of node-analyzer (since we bump it):
```
 ➜ helm dep update charts/sysdig-deploy
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "1password" chart repository
...Successfully got an update from the "longhorn" chart repository
...Successfully got an update from the "openebs" chart repository
...Successfully got an update from the "crowdsec" chart repository
...Successfully got an update from the "sysdig" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Error: can't get a valid version for repositories node-analyzer, kspm-collector. Try changing the version constraint in Chart.yaml
```
After a manual bump of `sysdig-deploy` the test is not failing neither locally or on GH pipeline
```
 ➜ helm unittest charts/sysdig-deploy

### Chart [ sysdig-deploy ] charts/sysdig-deploy

 PASS  Testing pre-generated values and best-known edge cases	charts/sysdig-deploy/tests/golden_template_test.yaml
 PASS  Test links in the notes section for regions	charts/sysdig-deploy/tests/notes_test.yaml
 PASS  Testing install commands in Readme	charts/sysdig-deploy/tests/readme_command_test.yaml
 PASS  Testing cluster scanner and runtime scanner should not be both enabled	charts/sysdig-deploy/tests/scannerconstraint_test.yaml

Charts:      1 passed, 1 total
Test Suites: 4 passed, 4 total
Tests:       65 passed, 65 total
Snapshot:    0 passed, 0 total
Time:        1.74095225s
```

**Review by chen-shmilovich-sysdig — APPROVED** [review]
LGTM

File: charts/node-analyzer/README.md
Comment (mavimo) [inline charts/node-analyzer/README.md:1]: Minor issue, we have some misalignment on the table

**Review by AlbertoBarba — APPROVED** [review]
